test/TEST-58-REPART/test.sh

```diff
@@ -15,6 +17,8 @@ test_append_files() {
         fi
         instmods dm_verity =md
         generate_module_dependencies
+        image_install -o /sbin/mkfs.btrfs
```

Comment:
Try to call `install_btrfs` (from `test/test-functions`) here instead of instead of just installing mkfs.btrfs - I suspect you're missing the udev rules hence the test fails.

```suggestion
        command -v mkfs.btrfs >/dev/null && install_btrfs
```